llvmlite/tests/test_binding.py

```diff
@@ -982,5 +982,47 @@ def test_libm(self):
         llvm.load_library_permanently(libm)
 
 
+class TestGlobalVariables(BaseTest):
+    def check_global_variable_linkage(self, linkage):
+        mod = ir.Module()
```

Comment:
Why are those tests going into the bindings test? IMHO it would be better to have them in the IR tests (and comparing the actual IR output as we do for other IR constructs).